docs/spec2006-tutorial.md

```diff
@@ -0,0 +1,517 @@
+# Tutorial: Run SPEC CPU 2006 Benchmarks in Full System Mode with gem5art  
+
+## Introduction  
+In this tutorial, we will demonstrate how to utilize gem5art to run SPEC CPU 2006 benchmarks in gem5 full system mode. 
+The scripts in this tutorial work with gem5art-* v0.3.1.
+
+### SPEC CPU 2006 Benchmarks  
+**Important:** The usage of this tutorial is just for the purpose of demonstration. 
+Those benchmarks [have been retired](https://www.spec.org/cpu2006/) [1]. 
+More details about those benchmarks are [here](https://dl.acm.org/citation.cfm?id=1186737) [2].  
+
+Examples of the usage of SPEC CPU 2006 benchmark in the literature are [here](https://ieeexplore.ieee.org/abstract/document/4378787) [3] and in the book [4]. 
+Another example of the usage of SPEC CPU benchmarks is [here](https://cacm.acm.org/magazines/2019/2/234352-a-new-golden-age-for-computer-architecture/fulltext) [5].
+
+### gem5 Full System Mode  
+Different from the gem5 SE (syscall emulation) mode, the full system mode uses the Linux system calls. 
+Therefore, the results would be more realistic if system calls are a significant portion of the benchmarks. 
+In order to run gem5 in the full system mode, gem5 requires a built Linux kernel, which is configurable. 
+gem5 does not support all configurations in Linux, but we will provide a Linux configuration that works with gem5. 
```

Comment:
`See the [run exit tutorial](THE PATH) for details on what kernels are currently tested with gem5.`